main.c

```diff
@@ -4,16 +4,18 @@
 
 #define GAME_INTRODUCTION_STATE 0
 #define MENU_STATE 1
-#define START_GAME_STATE 2
-#define PLAY_GAME_STATE 3
-#define WIN_STATE 4
-#define LOSE_STATE 5
+#define INSTRUCTION_STATE 2
+#define START_GAME_STATE 3
+#define PLAY_GAME_STATE 4
+#define WIN_STATE 5
+#define LOSE_STATE 6
 
 void ncursesInit();
 void ncursesEnd();
 int getRandomIntegerInRange(int min,int max);
 bool checkLoseCondition();
 bool checkWinCondition();
+bool isDown = false;
```

Comment:
That name is not meaningful. Could you rename it? Could be isUsingStatisticInstructions...